ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```diff
@@ -3547,6 +3558,1102 @@ router bfd
+   !
+   address-family flow-spec ipv4
+      bgp missing-policy direction in action deny-in-out
+      bgp missing-policy direction out action deny
+      neighbor FOOBAR activate
+      no neighbor IPv4_SEC activate
+      neighbor 192.168.66.22 activate
+   !
+   address-family flow-spec ipv6
+      bgp missing-policy direction in action deny
+      bgp missing-policy direction out action permit
+      no neighbor FOOBAR activate
+      neighbor IPv6_SEC activate
+      neighbor 192.168.66.21 activate
+   !
+   address-family ipv4
+      bgp additional-paths install
+      bgp additional-paths receive
+      bgp additional-paths send ecmp limit 20
+      no neighbor EVPN-OVERLAY-PEERS activate
+      neighbor foo additional-paths receive
+      neighbor foo prefix-list PL-BAR-v4-IN in
+      neighbor foo prefix-list PL-BAR-v4-OUT out
+      neighbor foo default-originate route-map RM-FOO-MATCH always
+      neighbor foo additional-paths send ecmp limit 20 prefix-list PL1
+      neighbor FOOBAR activate
+      neighbor FOOBAR next-hop address-family ipv6 originate
+      neighbor IPV4-UNDERLAY activate
+      neighbor IPV4-UNDERLAY route-map RM-HIDE-AS-PATH in
+      neighbor IPV4-UNDERLAY route-map RM-HIDE-AS-PATH out
+      no neighbor IPV4-UNDERLAY additional-paths send
+      neighbor IPv4-UNDERLAY-PEERS activate
+      neighbor MLAG-IPv4-UNDERLAY-PEER activate
+      neighbor OBS_WAN activate
+      neighbor OBS_WAN additional-paths send limit 8
+      neighbor SEDI activate
+      neighbor SEDI route-map RM-BGP-EXPORT-DEFAULT-BLUE-C1 out
+      neighbor SEDI-shut activate
+      neighbor SEDI-shut route-map RM-BGP-EXPORT-DEFAULT-BLUE-C1 out
+      neighbor TEST_PEER_GRP activate
+      neighbor TEST_PEER_GRP next-hop address-family ipv6 originate
+      neighbor TEST_RCF rcf in Address_Family_IPV4_In()
+      neighbor TEST_RCF rcf out Address_Family_IPV4_Out()
+      neighbor WELCOME_ROUTERS activate
+      neighbor WELCOME_ROUTERS additional-paths send any
+      neighbor 10.2.3.8 rcf in Address_Family_IPV4_In()
+      no neighbor 10.2.3.8 additional-paths send
+      neighbor 10.2.3.9 rcf out Address_Family_IPV4_Out()
+      neighbor 10.2.3.9 default-originate route-map Address_Family_IPV4 always
+      neighbor 10.2.3.9 additional-paths send ecmp limit 4
+      neighbor 192.0.2.1 additional-paths receive
+      neighbor 192.0.2.1 route-map Address_Family_IPV4_In in
+      neighbor 192.0.2.1 route-map Address_Family_IPV4_Out out
+      neighbor 192.0.2.1 prefix-list PL-FOO-v4-IN in
+      neighbor 192.0.2.1 prefix-list PL-FOO-v4-OUT out
+      neighbor 192.0.2.1 additional-paths send limit 20 prefix-list PL1
+      no neighbor 192.168.66.21 activate
+      neighbor 192.168.66.21 additional-paths send any
+      network 10.0.0.0/8
+      network 172.16.0.0/12
+      network 192.168.0.0/16 route-map RM-FOO-MATCH
+      no bgp redistribute-internal
+      redistribute attached-host route-map RM_BGP_EVPN_IPV4
+      redistribute bgp leaked
```

Comment:
Can we replace this one with host2's `redistribute bgp leaked route-map RM_BGP_EVPN_IPV4` from under the same AF?

Reply:
updated